src/components/Thumbnail/Thumbnail.scss

```diff
@@ -39,5 +39,5 @@
   max-width: 100%;
   max-height: 100%;
   @include recolor-icon(currentColor);
-  color: var(--p-icon, color('ink', 'lighter'));
+  color: var(--p-icon-subdued);
```

Comment:
This is on master. Do we still need that fallback for when new DL is off?

Reply:
Should be for both. Thanks for checking Ben!